spec/ParseFile.spec.js

```diff
@@ -1368,7 +1368,7 @@ describe('Parse.File testing', () => {
       await reconfigureServer({
         fileUpload: {
           enableForPublic: true,
-          fileExtensions: ['jpg'],
+          fileExtensions: ['jpg', 'any'],
```

Comment:
Could you use `mp3` instead and add a another file upload with a MP3 file, that is expected to succeed? MP3 is a completely different content type (not `image/...`). This makes the test more robust and verifies it works with 2 extensions of different content category.

```suggestion
          fileExtensions: ['jpg', 'mp3'],
```
of

Reply:
OK, but what content should I provide? I don't think you want really mp3 encoded in base64.

Reply:
Then maybe let's use a WAV file, it's shorter in base64 encoding:

`data:audio/wav;base64,UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA`

Reply:
Is it OK now?

Reply:
Yes!